content/en/docs/ops/integrations/spire/index.md

```diff
@@ -27,180 +27,145 @@ The integration is compatible with Istio upgrades.
+{{< /tip >}}
 
-See the [SPIRE's Quick start for Kubernetes guide](https://spiffe.io/docs/latest/try/getting-started-k8s/)
-to get started deploying SPIRE into your Kubernetes environment. See [SPIRE CA Integration Prerequisites](#spire-ca-integration-prerequisites)
-for more information on configuring SPIRE to integrate with Istio deployments.
 
-#### SPIRE CA Integration Prerequisites
+This will also install 
 
-To integrate your SPIRE deployment with Istio, configure SPIRE:
+- The [SPIFFE CSI driver](https://github.com/spiffe/spiffe-csi) which is used to mount an Envoy-compatible SDS socket into proxies. Using the SPIFFE CSI driver to mount SDS sockets is strongly recommended by both Istio and SPIRE, as `hostMounts` are a larger security risk and introduce operational hurdles.
 
-1. Access the [SPIRE Agent reference](https://spiffe.io/docs/latest/deploying/spire_agent/#agent-configuration-file) and
-    configure the SPIRE Agent socket path to match the Envoy SDS defined socket path.
+- The [SPIRE Controller Manager](https://github.com/spiffe/spire-controller-manager), which eases the creation of SPIFFE registrations for workloads.
 
-    {{< text plain >}}
-    socket_path = "/run/secrets/workload-spiffe-uds/socket"
-    {{< /text >}}
+## Register workloads
 
-1. Share the SPIRE Agent socket with the pods within the node by deploying the
-    [SPIFFE CSI Driver](https://github.com/spiffe/spiffe-csi).
-    The `-workload-api-socket-dir` argument to the driver should be the mount location of the socket's directory.
+SPIRE by-design only grants identities to workloads that have been registered with the SPIRE server.
 
-See [Install Istio](#install-istio) to configure Istio to integrate with the SPIFFE CSI Driver.
+This includes your user workloads, as well as Istio's own workloads - Istio sidecars and gateways, once configured for SPIRE integration, cannot get identities, and therefore cannot reach READY status, unless there is a preexisting, matching SPIRE registration created for them ahead of time.
 
-{{< tip >}}
-Istio will become the Envoy SDS listener if the socket is not created by SPIRE before the Istio agent starts up. This timing is controlled by customizing the IstioOperator.
-{{< /tip >}}
+See the [SPIRE help on Registering workloads](https://spiffe.io/docs/latest/deploying/registering/) for more information on using multiple selectors to strengthen attestation criteria, and the selectors available.
 
-## Install Istio
+This section describes the options available for registering Istio workloads in a SPIRE Server and provides some example workload registrations.
 
-### Option 1: Configuration for Workload Registration with the SPIRE Controller Manager
+{{< warning >}}
+Istio currently requires a specific SPIFFE ID format for workloads. All registrations must follow the Istio SPIFFE ID pattern: `spiffe://<trust.domain>/ns/<namespace>/sa/<service-account>`
```

Comment:
Currently?  Is this expected to change?

Reply:
We have multiple open bugs to fix this and I intend to change it a bit, so yes.